rmgpy/data/thermo.py

```diff
@@ -123,14 +123,17 @@ def save_entry(f, entry):
             f.write('        CpInf = {0!r},\n'.format(entry.data.CpInf))
         f.write('    ),\n')
     else:
-        f.write('    thermo = {0!r},\n'.format(entry.data))
+        if entry.data:
+            f.write('    thermo = u{0!r},\n'.format(entry.data))
```

Comment:
Either that or this might be causing things to break? The `u` might be causing things to break.